packages/reactivue/package.json

```diff
@@ -9,6 +9,11 @@
   "browser": "dist/index.esm.js",
   "types": "dist/index.d.ts",
   "exports": {
+    ".": {
+      "import": "./dist/index.esm.js",
+      "require": "./dist/index.js",
+    },
+    "./": "./",
```

Comment:
What's this for? Exposing everything under the package?

Reply:
tbh i dont really know, just saw it on https://github.com/developit/htm/blob/master/package.json